lib/plugins/wallet/zcashd/zcashd.js

```diff
@@ -151,11 +151,18 @@ function checkBlockchainStatus (cryptoCode) {
     .then(res => !!res['initial_block_download_complete'] ? 'ready' : 'syncing')
 }
 
+function getTxHashesByAddress (cryptoCode, address) {
+  checkCryptoCode(cryptoCode)
+    .then(() => fetch('getaddresstxids', [address]))
+    .then(_.values)
+}
```

Comment:
ZEC 5.1.0 documentation shows that the `getaddresstxids` already returns an array with the transaction IDs, so there shouldn't be need for `_.values` https://zcash.github.io/rpc/getaddresstxids.html

```suggestion
function getTxHashesByAddress (cryptoCode, address) {
  checkCryptoCode(cryptoCode)
    .then(() => fetch('getaddresstxids', [address]))
}
```